frontend/src/core/components/AuthSidebar/AuthSidebar.scss

```diff
@@ -2,18 +2,37 @@
   margin-top: 35%;
 
   h1 {
-    color: #ffffff;
+    width: 254px;
+    height: 90px;
+    font-family: OpenSans;
+    font-size: 30px;
+    font-weight: bold;
+    font-stretch: normal;
+    font-style: normal;
     line-height: 1;
-    margin-bottom: 25px;
-    font-weight: 700;
+    letter-spacing: normal;
+    color: #ffffff;
   }
 
   p {
     font-size: 14px;
     letter-spacing: -0.38px;
+    color: #ffffff;
+    font-family: Lato;
```

Comment:
About this properties like font we have a scss file, that has font properties, please import that file(Variables/Font.scss) and use it here.